apps/shipments/serializers.py

```diff
@@ -203,8 +205,16 @@ def update(self, instance, validated_data):
 
     def validate_device_id(self, device_id):
         auth = self.context['auth']
+        shipment = self.context['shipment']
+        if not device_id or device_id == 'null':
+            if not shipment.device:
+                return None
+            if not shipment.delivery_act or shipment.delivery_act >= datetime.utcnow().replace(tzinfo=pytz.UTC):
```

Comment:
Use `datetime.now(timezone.utc)` instead of `datetime.utcnow().replace(tzinfo=pytz.UTC)`